GCRCatalogs/catalog_configs/dc2_metacal_run2.1.1i.yaml

```diff
@@ -1,4 +1,8 @@
 subclass_name: dc2_metacal.DC2MetacalCatalog
 base_dir: /global/projecta/projectdirs/lsst/production/DC2_ImSim/Run2.1.1i/dpdd/calexp-v1:coadd-v1/metacal_table_summary/final
+schema_filename: schema.yaml
```

Comment:
Do we still need this? I believe this is for hdf5 files only?

Reply:
maybe not... yes, I think you are right

Reply:
better :-) ?